redis/connection.py

```diff
@@ -493,6 +502,20 @@ def read_response(self):
     DefaultParser = PythonParser
 
 
+class ClusterParser(DefaultParser):
```

Comment:
IMHO this really belongs in redis/cluster.py. Since it's only used there, it keeps the cluster code nicely self-contained.

Reply:
agreed, moved to cluster.py.